src/m365/aad/commands/user/user-get.ts

```diff
@@ -50,7 +66,20 @@ class AadUserGetCommand extends GraphCommand {
 
     request
       .get(requestOptions)
-      .then((res: any): void => {
+      .then((res: any): Promise<User> => {
+        if (args.options.email) {
+          if (res.value.length > 0) {
+            return Promise.resolve(res.value[0]);
```

Comment:
Rather than returning the first user silently, it would be better to throw an error and prompt the user to disambiguate using the user name or ID. Otherwise you might end up working with the wrong user object.